### PR TITLE
Set sonar values needed by the pipeline

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -91,6 +92,18 @@ jacocoTestReport {
   }
 }
 
+project.tasks['sonarqube'].dependsOn test, integration
+
+sonarqube {
+  properties {
+    property "sonar.projectName", "Reform :: spring-boot-template"
```
Comment (doncem) [inline build.gradle]: We could un-slugify the name

File: build.gradle [hunk 2]
```diff
@@ -91,6 +92,18 @@ jacocoTestReport {
...
+    property "sonar.projectKey", "uk.gov.hmcts.reform.demo:spring-boot-template"
```
Comment (doncem) [inline build.gradle]: Perhaps `demo` can be skipped here, don't you think?

File: build.gradle (hunk 1 above)
Comment (timja) [reply build.gradle]: what do you mean?

File: build.gradle (hunk 2 above)
Comment (timja) [reply build.gradle]: sure

File: build.gradle (hunk 1 above)
Comment (doncem) [reply build.gradle]: Ah true. That buggy init script

**Pull request merged**